go.mod

```diff
@@ -3,50 +3,53 @@ module github.com/filecoin-project/mir
 go 1.18
 
 require (
-	github.com/AlecAivazis/survey/v2 v2.3.2
+	github.com/AlecAivazis/survey/v2 v2.3.5
 	github.com/alecthomas/template v0.0.0-20190718012654-fb15b899a751 // indirect
 	github.com/alecthomas/units v0.0.0-20211218093645-b94a6e3cc137 // indirect
-	github.com/libp2p/go-libp2p v0.19.3
-	github.com/libp2p/go-libp2p-core v0.15.1
+	github.com/libp2p/go-libp2p v0.21.0
```

Comment:
Please don't change libp2p dependencies. 
They must be the same as in Eudico - https://github.com/filecoin-project/eudico/blob/eudico/go.mod#L115